.github/workflows/lint.yml

```diff
@@ -104,6 +104,12 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0 # deep clone, to allow us to use git merge-base
+      - name: Fetch PR diff
+        env:
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+        run: |
+          cd "${GITHUB_WORKSPACE}"
+          wget -O pr.diff "https://patch-diff.githubusercontent.com/raw/pytorch/pytorch/pull/$PR_NUMBER.diff"
```

Comment:
Is there a rate limit for this type of stuff? I feel like we would probably hit it with our traffic

Reply:
I don't think so, we've been using it in prod for a bit: https://github.com/pytorch/pytorch/blob/master/.github/workflows/lint.yml#L346

We should probably use the public url and follow redirects instead though, so something like `https://github.com/pytorch/pytorch/pull/63550.diff`